maintainer/channels.py

```diff
@@ -0,0 +1,145 @@
+import requests
+import configuration
+
+
+class ChannelsMaintanance(object):
+    '''
+    Class is using Teamspeak API in order to maintain channels
+    which are not used.
+    '''
+    def __init__(self, qtime=2):
+        '''
+        Input:
+            qtime = time in seconds after which we are taking steps:
```

Comment:
Do you really want to keep qtime in seconds? I think that days or even hours would be more suitable for this usage, as we want to keep channels for days by default, not for seconds.

Reply:
fixed, changed to days